### PR TITLE
add deb and rpm building

### PR DESCRIPTION
**Pull request by lexelby**
Easy peasy!  I think.

### CODE REVIEW

File: bin/linux-sh-installer [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  # allow expert user override
+  if [ -n "$INKSCAPE_EXTENSIONS_PATH" ]; then
+    echo "$INKSCAPE_EXTENSIONS_PATH"
+    return
+  fi
+
+  inkscape="$(find_inkscape)"
+
+  if [ -x "$inkscape" ]; then
+    extensions_dir="$(inkscape --user-data-directory)"
```
Comment (kaalleen) [inline bin/linux-sh-installer:69]: This is the inkscape folder not the extensions folder. But it is used as the extensions folder. I think we should add /extensions here.

**Comment by kaalleen** [comment]
The rpm package is complaining about being ... unsinged ... (OpenSuSe). That's unpleasant news :( Well, other than macOS users can still click on ignore. Fedora complains that it is unsafe in the summary before installation.

deb package tested on Ubuntu 20.04.3 LTS  (virtual environment) claims that it cannot install inkstitch because the version number doesn't start with a digit.

Just to keep track on it. Fedora is showing the square issue in params. OpenSuSe doesn't. Unsure if fc-config has been cleared in this SuSe version or not.

File: bin/linux-sh-installer (hunk 1 above)
Comment (lexelby) [reply bin/linux-sh-installer:69]: Oh shoot, thank you!

**Comment by lexelby** [comment]
Oops, thanks for fixing my brain-o!

**Comment by lexelby** [comment]
The version number thing is easy enough.  I'll push a commit to fix that.  That's just for dev versions.

**Comment by kaalleen** [comment]
rpm is still in trouble: Signature verification failed [4-Signatures public signature key is not available]

**Review by kaalleen — APPROVED** [review]

**Pull request merged**